virtualmachine/vsphere/util.go

```diff
@@ -63,12 +63,33 @@ func StringInSlice(str string, list []string) bool {
 	return false
 }
 
+// getVMSearchFilter: returns VMSearchFilter object for given vm
+// by default vm is searched within DC
+func getVMSearchFilter(vm *VM) VMSearchFilter {
```

Comment:
Change this to accept the name of the vm instead, that way i can search any vm not just vm.Name.